tasks-contribution-guide.md

```diff
@@ -17,4 +17,17 @@ We have `tasks/assets` that contains data used in the inference widget and image
 
 ![Libraries of a Task](tasks/assets/contribution-guide/libraries.png)
 
-This might seem overwhelming, but you don't necessarily need to add all of these in one pull request or on your own, you can simply contribute one section. Feel free to ask for help whenever you need. 
\ No newline at end of file
+This might seem overwhelming, but you don't necessarily need to add all of these in one pull request or on your own, you can simply contribute one section. Feel free to ask for help whenever you need. 
+
+### I have a merge conflict, what now! 😱
+So essentially what happens is that you were trying to write your changes to this repository's main branch. However, since you opened the PR, some changes occurred to that branch. Because of this, there's a conflict. Git says that "for fifth line of the code you wanted to write this, but this line is now changed, so it's better if you adapt your changes on top of these changes".
```

Comment:
```suggestion
So essentially, what happens is that you were trying to write your changes to this repository's main branch. However, since you opened the PR, some changes occurred to that branch in the same files you're modifying. Because of this, there's a conflict. Git says that "for fifth line of the code you wanted to write this, but this line is now changed, so it's better if you adapt your changes on top of these changes".
```